kafka-connect-s3/src/test/java/io/confluent/connect/s3/storage/S3OutputStreamTest.java

```diff
@@ -53,4 +67,77 @@ public void testPropagateOtherRetriableS3Exceptions() {
     when(s3Mock.initiateMultipartUpload(any())).thenThrow(new AmazonClientException(S3_EXCEPTION_MESSAGE));
     assertThrows(IOException.class, () -> stream.commit());
   }
+
+  @Test
+  public void testNewMultipartUploadAESSSE() throws IOException {
```

Comment:
Is AES with customer key, a valid case? If yes, we should add a test for it as well.